util/reggen/ip_block.py

```diff
@@ -19,6 +20,45 @@
 from reggen.signal import Signal
 from reggen.countermeasure import CounterMeasure
 
+# Known unique comportable IP names and associated CIP_IDs.
+KNOWN_CIP_IDS = {
```

Comment:
@cfrantz given that this is a unique name to number mapping, and given that we do not expose CIP_IDs in registers anymore, do we even need the CIP_ID key in the Hjson? It almost feels like it is redundant at this point.

Reply:
I would argue that we should throw CIP IDs out (or at least not require them). Requiring IDs in an in-tree table prevents out-of-tree use of the tools.